src/Farmer/Builders/Builders.Functions.fs

```diff
@@ -12,7 +12,10 @@ open Farmer.Arm.KeyVault
 open Farmer.Arm.KeyVault.Vaults
 open System
 
-type FunctionsRuntime = DotNet | Node | Java | Python
+type RuntimeKind =
+    | Isolated
+    | InProcess
+type FunctionsRuntime = DotNet of RuntimeKind | Node | Java | Python
```

Comment:
Yes, that's a good point, `use_runtime DotNetIsolated` feels better than `use_runtime (DotNet Isolated)`. 

Since the RuntimeKind was an idea from @isaacabraham maybe he wants to weigh in as well? Should we just go with a `DotNetIsolated` case?

Reply:
Yes, I think merging them will be better from a public API consumption point of view.